pkg/chartutil/create.go

```diff
@@ -59,6 +61,18 @@ service:
   type: ClusterIP
   externalPort: 80
   internalPort: 80
+ingress:
+  enabled: false
+  # Used to create Ingress record (should used with service.type: ClusterIP).
+  # hostname: chart-example.local
```

Comment:
@archSeer I agree in most cases I'd use a single host, but also agree are many times where multiple hosts are valuable for an application - so an array of hostnames is a good standard (if a specific chart's application only allows a single hostname for some reason it can add a single item to the array and follow the standard. Most apps do not have such a restriction). I'll update that in this PR. Also if `hosts` is more standard, I'll change that key to match.